petab/visualize/plot_data_and_simulation.py

```diff
@@ -100,7 +100,7 @@ def plot_data_and_simulation(
         # import from file
         exp_data = measurements.get_measurement_df(exp_data)
         # check columns, and add non-mandatory default columns
-        exp_data, dataset_id_list, legend_dict = \
+        exp_data, _, legend_dict = \
```

Comment:
could you explain this change? is dataset_id_list not necessary anymore?

Reply:
I think it never was in reasonable cases:
* if you hand over to plot_data_and_simulation a dataset_id_list of ids you want to plot, then this list is left untouched by the check_ex_exp_col function
* if you hand over something else like cond_id_list to plot_data_and_simulation then you do not have a visu_df. In line 103 then exp_data gets a new dataset_id column which is also needed. Later in the code a virtual visu_df is created with a dataset_id column made in the same way
* if you provide visu_df than it hast mandatory the dataset_id column and so should mes_df and sim_df have

Reply:
thanks for the explanation! okay, the only thing I'm not sure at the moment, I think we decided that dataset_id column is not mandatory anymore, also if you provide visu_df... but I dont know if this was already addressed.... but maybe this is then another issue

Reply:
ok, you are right. It is not mandatory anymore. Thanks, I missed that. I will change the code.